### DISCUSSION TITLE
Table cell avatar content bug

### DISCUSSION THREAD

**Discussion by karoltonik** [twilio-labs/paste#3433] [Help: Figma] [answered]
I've noticed that when changing the variant in a table cell to "avatar" and later switching the content type for the avatar in the table cell content from "image" to "initial" an additional square border appears that shouldn't be there. Interestingly, when changing the size of the avatar to any other value, the border disappears. If we then set the size back to the default 32px, the square border also disappears.

Po zmianie avatar content from image (default) to initial:
<img width="782" alt="image" src="https://github.com/twilio-labs/paste/assets/141224835/06800847-fa7f-415f-a663-a1b3683c67e5">

**Comment by serifluous** [comment, marked as answer]
Hey @karoltonik, we have a fix for this in our component library. I'm just waiting on Figma to resolve some publishing issues on our file. I'll reply here when the fix is published.

**Reply by serifluous** [reply]
The fix has just been published in our Figma library!
